time stamp
float32 range
uint16 samples_size     #number of samples in the array
uint16[] samples
